marklogic-data-hub-central/gradle.properties

```diff
@@ -2,8 +2,8 @@ reactUiPath=./ui
 springBootUiPath=src/main/resources/static
 springBootJarName=marklogic-data-hub-central
 
-springBootVersion=2.2.6.RELEASE
-springSecurityVersion=5.2.3.RELEASE
+springBootVersion=2.2.8.RELEASE
+springSecurityVersion=5.1.11.RELEASE
```

Comment:
This seems off - we're moving to a lower version of Spring Security?

Reply:
That was the version bump intended for marklogic-data-hub core. I've adjusted.